README.md

```diff
@@ -21,6 +21,12 @@ https://github.com/ykdojo/kaguya/assets/107422421/d61b8ff1-2dbd-4eb4-b1b5-45d437
 1. If you want Kaguya to be able to interact with your files, put them in the FILES folder.
 1. Note: Kaguya won't have access to files outside of its own directory.
 
+## Recommended Custom Instructions
+
+1. When editing a file of 50+ lines with Kaguya, use search and replace, not ```updateWholeFile```. Confirm with me before editing a file.
```

Comment:
I think these should be in a code block format and ready for copy & pasting - what do you think?

Reply:
That would be better, I'll make changes and send back.

Reply:
Were you able to view the second commit ok? I changed to block format for easy copy and pasting and not sure if I re-pushed the pull request correctly the second time around. Let me know thanks!